libtracepoint-control-cpp/include/tracepoint/TracepointSession.h

```diff
@@ -644,7 +730,7 @@ namespace tracepoint_control
         - ENOMEM: memory allocation failed.
         */
         _Success_(return == 0) int
-        DisableTracePoint(unsigned id) noexcept;
+        DisableTracepoint(unsigned id) noexcept;
```

Comment:
Note: Breaking change. The Linux perf community and the rest of the LinuxTracepoints repo use a lowercase P.